cat_emails/email_scanner_consumer.py

```
@@ -10,25 +10,92 @@
 from imapclient import IMAPClient
 from email import message_from_bytes
 from bs4 import BeautifulSoup
+from typing import Dict, Optional, Tuple, Callable
```

Ruff (0.8.0)

`typing.Dict` imported but unused

Remove unused import

(F401)